GOVERNANCE.md

```diff
@@ -0,0 +1,33 @@
+# Governance
+
+## Project Maintainers
+
+[Project maintainers](CODEOWNERS) are responsible for all activities
+maintaining and updating the Notary project. Final decisions on the projects
+reside with the project maintainers.
+
+Maintainers MUST remain active. To remain active they MUST collaborate on the 
+project at least once in a 3 month period. Example collaborations include, attending
+the community meeting, creation or commenting on issues/PRs, PR reviews, etc. 
+If they are unresponsive for >3 months, they will be automatically moved to emeritus unless a
+[super-majority](https://en.wikipedia.org/wiki/Supermajority#Two-thirds_vote) of
+the other project maintainers agrees to extend the period to be greater than 3
+months.
+
+New maintainers can be added to the project by a
+[super-majority](https://en.wikipedia.org/wiki/Supermajority#Two-thirds_vote)
+vote of the existing maintainers. A potential maintainer may be nominated by an existing maintainer.
+Submissions for new maintainers can be made by creating an
+[issue](https://github.com/notaryproject/notaryproject/issues/new). The current maintainers will be able to cast
+a single vote each via a +1 comment on the issue. Once a [super-majority](https://en.wikipedia.org/wiki/Supermajority#Two-thirds_vote)
```

Comment:
> The current maintainers will be able to cast
> a single vote each via a +1 comment on the issue.

Does this in practice mean that two votes in total (one by each current maintainer)? If so, both maintainers must cast their votes for the exact same nominee for there to be anyone elected. Or is it, each maintainer can vote just once for every nominee?

Reply:
The latter. This could change if we prefer the idea of having a fixed number of maintainer seats, then we could do a ranked voting across candidates.